tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -2503,7 +2541,7 @@ public function testMergeWithFieldList()
         $entity->accessible('*', false);
         $entity->isNew(false);
         $entity->clean();
-        $result = $marshall->merge($entity, $data, ['fieldList' => ['title', 'body']]);
+        $result = $marshall->merge($entity, $data, [$fields => ['title', 'body']]);
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?